gems/quilt_rails/test/controllers/quilt/ui_controller_test.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+require 'rails/generators'
+require 'generators/quilt/rails_setup/rails_setup_generator'
+
+module Quilt
+  class UiControllerTest < ActionDispatch::IntegrationTest
+    def run(*)
```

Comment:
`include ActiveSupport::Testing::Isolation`

All tests needs in this file needs to run in a fork, otherwise as soon as someone adds another test and call `boot_dummy` subsequent tests would fail

Reply:
Ah! I was going to ask about what rails does to solve this problem. Thank you 🙇